### PR TITLE
Global toleration and affinity

### PR DESCRIPTION
**Pull request by oldgiova**
Ticket: QA-583

Added default toleration and affinity for `mender-helm-test`. With the `merge` strategy, values can be overridden by specific values.

### CODE REVIEW

File: mender/templates/api-gateway-deploy.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {{- if .Values.api_gateway.enabled }}
+{{- $merged := merge (deepCopy .Values.api_gateway) (deepCopy (default (dict) .Values.global)) -}}
```
Comment (alfrunes) [inline mender/templates/api-gateway-deploy.yaml:2]: Should we instead create a new `default` section? The `global` values are shared across sub-charts and already contain a lot of attributes that are not used at the service level.

Comment (alfrunes) [reply mender/templates/api-gateway-deploy.yaml:2]: I would prefer the first option to avoid potential name collisions.

Comment (oldgiova) [reply mender/templates/api-gateway-deploy.yaml:2]: Got it. Let me fix

Comment (oldgiova) [reply mender/templates/api-gateway-deploy.yaml]: Done, you can review it again

**Review by alfrunes — APPROVED** [review]

**Pull request merged**